ipaserver/install/bindinstance.py

```diff
@@ -805,6 +805,8 @@ def __setup_sub_dict(self):
             NAMED_VAR_DIR=paths.NAMED_VAR_DIR,
             BIND_LDAP_SO=paths.BIND_LDAP_SO,
             INCLUDE_CRYPTO_POLICY=crypto_policy,
+            DATA_DIR='' if NAME == 'debian' else 'data/',
+            DISABLE_DEBIAN='//' if NAME == 'debian' else '',
```

Comment:
Please handle platform-specific code in the `ipaplatform` module.

Reply:
ok, this was reviewed earlier in https://github.com/freeipa/freeipa/pull/673 so I just slapped it in here